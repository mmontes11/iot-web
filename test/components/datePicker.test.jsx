import React from "react";
import { shallow } from "enzyme";
import DatePicker from "components/datePicker";
import IntlProvider from "containers/intlProvider";
import store from "config/store";

describe("components/datePicker", () => {
  it("renders in initial state", () => {
    const wrapper = shallow(
      <IntlProvider store={store}>
        <DatePicker placeholder="Select date:" selected={undefined} onChange={() => undefined} />
      </IntlProvider>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
