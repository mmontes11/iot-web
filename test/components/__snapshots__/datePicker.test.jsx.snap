// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/datePicker renders in initial state 1`] = `
<IntlProvider
  dispatch={[Function]}
  locale="en"
  messages={
    Object {
      "Custom": "Custom",
      "Data": "Data",
      "Delete": "Delete",
      "Error": "Error",
      "Event stats": "Event stats",
      "Events:": "Events:",
      "Filters": "Filters",
      "Google maps": "Google maps",
      "Group by": "Group by",
      "IP address:": "IP address:",
      "IoT": "IoT",
      "Last observation:": "Last observation:",
      "Login": "Login",
      "Logout": "Logout",
      "MQTT topic:": "MQTT topic:",
      "Measurement stats": "Measurement stats",
      "Measurements:": "Measurements:",
      "Password": "Password",
      "Request failed": "Request failed",
      "Reset": "Reset",
      "Select end date": "Select end date",
      "Select observation": "Select observation",
      "Select start date": "Select start date",
      "Select thing": "Select thing",
      "Select time period": "Select time period",
      "Select type": "Select type",
      "Stats": "Stats",
      "Thing not found": "Thing not found",
      "Things": "Things",
      "Time": "Time",
      "Time period": "Time period",
      "Username": "Username",
      "avg": "Average",
      "avgByHour": "Average by hour",
      "date": "Date",
      "day": "Day",
      "door-closed": "Door closed",
      "door-opened": "Door opened",
      "en": "English",
      "es": "Spanish",
      "event": "Event",
      "hour": "Hour",
      "humidity": "Humidity",
      "humidity-indoor": "Humidity indoor",
      "humidity-outdoor": "Humidity outdoor",
      "max": "Maximum",
      "maxByHour": "Maximum by hour",
      "measurement": "Mesasurement",
      "min": "Minimum",
      "minByHour": "Minimum by hour",
      "minute": "Minute",
      "month": "Month",
      "second": "Second",
      "stdDev": "Standard deviation",
      "stdDevByHour": "Standard deviation by hour",
      "temperature": "Temperature",
      "temperature-indoor": "Temperature indoor",
      "temperature-outdoor": "Temperature outdoor",
      "thing": "Thing",
      "total": "Total",
      "week": "Week",
      "year": "Year",
    }
  }
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
  storeSubscription={
    Subscription {
      "listeners": Object {
        "clear": [Function],
        "get": [Function],
        "notify": [Function],
        "subscribe": [Function],
      },
      "onStateChange": [Function],
      "parentSub": undefined,
      "store": Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      },
      "unsubscribe": [Function],
    }
  }
>
  <InjectIntl(DatePicker)
    onChange={[Function]}
    placeholder="Select date:"
  />
</IntlProvider>
`;
